osgp/platform/osgp-adapter-domain-core/src/test/java/org/opensmartgridplatform/adapter/domain/core/application/services/ConfigurationManagementServiceTest.java

```diff
@@ -0,0 +1,275 @@
+import org.opensmartgridplatform.domain.core.valueobjects.RelayMap;
+import org.opensmartgridplatform.domain.core.valueobjects.RelayType;
+import org.opensmartgridplatform.dto.valueobjects.ConfigurationDto;
+import org.opensmartgridplatform.shared.exceptionhandling.ComponentType;
+import org.opensmartgridplatform.shared.exceptionhandling.FunctionalException;
+import org.opensmartgridplatform.shared.exceptionhandling.OsgpException;
+import org.opensmartgridplatform.shared.infra.jms.CorrelationIds;
+import org.opensmartgridplatform.shared.infra.jms.RequestMessage;
+import org.opensmartgridplatform.shared.infra.jms.ResponseMessage;
+import org.opensmartgridplatform.shared.infra.jms.ResponseMessageResultType;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+@ExtendWith(MockitoExtension.class)
+public class ConfigurationManagementServiceTest {
+    @Mock
+    private WebServiceResponseMessageSender webServiceResponseMessageSender;
+    @Mock
+    private SsldRepository ssldRepository;
+    @Mock
+    private ConfigurationDto configurationDto;
+    @Mock
+    private DomainCoreMapper domainCoreMapper;
+    @Mock
+    private OrganisationDomainService organisationDomainService;
+    @Mock
+    private DeviceDomainService deviceDomainService;
+    @Mock
+    private CorrelationIds correlationIds;
+    @Mock
+    private Configuration configuration;
+    @Mock
+    private OsgpCoreRequestMessageSender osgpCoreRequestMessageSender;
+    @Mock
+    private Ssld ssld;
+    @Mock
+    private Device device;
+    @Captor
+    private ArgumentCaptor<RequestMessage> requestMessageArgumentCaptor;
+    @Captor
+    private ArgumentCaptor<String> messageTypeArgumentCaptor;
+    @Captor
+    private ArgumentCaptor<String> ipAddressArgumentCaptor;
+    @Captor
+    private ArgumentCaptor<Integer> messagePriorityArgumentCaptor;
+    @Captor
+    private ArgumentCaptor<Long> scheduledTimeArgumentCaptor;
+    @Captor
+    private ArgumentCaptor<ResponseMessage> responseMessageArgumentCaptor;
+    @InjectMocks
+    @Qualifier("organisationDomainService")
+    private ConfigurationManagementService configurationManagementService;
+
+    private static final String organisationIdentification = "orgIdentification";
+    private static final String deviceIdentification = "deviceIdentification";
+    private static final String correlationUid = "correlationUid";
+    private static final long scheduleTime = 1;
+    private static final String messageType = "none";
+    private static final int messagePriority = 1;
+    private static final String ipAddress = "333.333.1.22";
+
+    private OsgpException exception;
+    private static final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+
+    @BeforeEach
+    public void setUp() throws NoSuchFieldException,
+            SecurityException, IllegalArgumentException, IllegalAccessException {
+        // make correlationIds
```

Comment:
A single-line comment should be preceded by a blank line, see https://www.oracle.com/java/technologies/javase/codeconventions-comments.html#341